programs/redstone-sol/src/state.rs

```diff
@@ -0,0 +1,37 @@
+use anchor_lang::prelude::*;
+
+pub type SignerAddress = [u8; 20];
+pub type FeedId = [u8; 32];
+
+#[account]
+#[derive(Default)]
+pub struct PriceData {
+    pub feed_id: FeedId,
+    pub value: u128,
+    pub timestamp: u64,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct DataPoint {
+    pub feed_id: FeedId,
+    pub value: u128,
```

Comment:
generally we use U256 for values (because the current format returns 32 bytes as a value size).

Reply:
1ff23264ddfb53005201a472edfebdca996243d5